server/graphql/queries.js

```diff
@@ -36,9 +36,9 @@ const queries = {
     }
   },
   /*
-   * Given a collective slug, returns the events
+   * Given a collective slug, returns all events
    */
-  getEvents: {
+  allEventsForCollective: {
```

Comment:
do we really need "ForCollective"?
I think that this is a case where we can overload `allEvents` with or without a `CollectiveSlug`, because in both cases we expect an array of events.